diff -ruN orig/src/jobs/meltjob.cpp new/src/jobs/meltjob.cpp
--- orig/src/jobs/meltjob.cpp	2020-11-28 11:58:25.000000000 -0600
+++ new/src/jobs/meltjob.cpp	2020-12-09 14:00:44.281331487 -0600
@@ -95,7 +95,7 @@
         return;
     }
     QString shotcutPath = qApp->applicationDirPath();
-    QFileInfo meltPath(shotcutPath, "melt");
+    QFileInfo meltPath(shotcutPath, "mlt-melt");
     setReadChannel(QProcess::StandardError);
     QStringList args;
     args << "-verbose";
